packages/cashier/src/hooks/__tests__/useCountdown.test.tsx

```diff
@@ -0,0 +1,156 @@
+import * as React from 'react';
+import { render, screen, waitFor } from '@testing-library/react';
```

Comment:
After upgrading to react 18 we should use [renderHook](https://testing-library.com/docs/react-testing-library/api#renderhook) instead.

Reply:
Please add this comment as `TODO` comment inside the code base.